### DISCUSSION TITLE
vue 图片加载

### DISCUSSION THREAD

**Discussion by zqm971210** [zxuqian/frontend-questions#9] [求助]
## 描述一下问题？
请问 up  vue 中 怎么才能监听 图片加载完成事件呢?

## 问题出现的场景、原因、截图和重现过程（适用于代码类）？
这是一张 2M 大小的图片
![image](https://user-images.githubusercontent.com/48125795/98620920-2d03c400-2341-11eb-97f3-be040ddcdae4.png)

我是这样展示图片的
![image](https://user-images.githubusercontent.com/48125795/98621062-6e946f00-2341-11eb-9831-f5ed98f096cc.png)

但是当我网络不好的时候 就会出现 上面图1的情况

加载完成的时候是这样的
![image](https://user-images.githubusercontent.com/48125795/98621292-e2cf1280-2341-11eb-808c-d33b5fdcdb1b.png)

我该如何获取图片加载完成的事件呢?

或者 up 有什么好的方法在我网络情况不好的时候解决这样的问题呢?

**Comment by zxuqian** [comment]
参考：https://stackoverflow.com/questions/280049/how-to-create-a-javascript-callback-for-knowing-when-an-image-is-loaded
